sdk/lib/3rdparty/freetype/freetype.rc

```diff
@@ -3,6 +3,6 @@
 #define REACTOS_STR_INTERNAL_NAME	"freetype\0"
 #define REACTOS_STR_ORIGINAL_FILENAME	"freetype.dll\0"
 #define REACTOS_STR_COMPANY_NAME	"ReactOS Development Team/FreeType Team\0"
-#define REACTOS_STR_LEGAL_COPYRIGHT	"Copyright 1998-2011 ReactOS Team\0"
-#define REACTOS_STR_ORIGINAL_COPYRIGHT	"Copyright 1996-2013 FreeType Team\0"
+#define REACTOS_STR_LEGAL_COPYRIGHT	"Copyright 1998-2025 ReactOS Team\0"
+#define REACTOS_STR_ORIGINAL_COPYRIGHT	"Copyright 1996-2019 FreeType Team\0"
```

Comment:
See my previous comment: this .rc file can be removed (freetype is a static library, not a dll); instead you should update the freetype.rc file in win32ss/drivers/font/ftfd.

Reply:
Fixed `freetype.rc` in `win32ss/drivers/font/ftfd`.